winter-website-contact-form.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE HTML>
+
+<html lang="en-GB">
+
+<head>
+	
+	<div class="navBar">
```

Comment:
the `<head>` tag is not for display HTML. the navbar should go in the body. the head is just for the title and css link tags